app/src/main/java/com/infomaniak/mail/ui/main/menu/ManageMailAddressFragment.kt

```diff
@@ -51,10 +54,20 @@ class ManageMailAddressFragment : Fragment() {
         AccountUtils.currentUser?.let { user ->
             avatar.loadAvatar(user)
             mail.text = user.email
+
+            dialogBinding.confirmLogoutDialogTitle.text = getString(R.string.confirmLogoutTitle, user.email)
```

Comment:
dialogBinding will be automatically loaded each time this view is displayed, even if the user never clicks on "Logout"

Reply:
Nice catch, I fixed it